src/utils/route.ts

```diff
@@ -0,0 +1,26 @@
+import { AppRoutes } from '@/config/routes'
+import type { ParsedUrlQuery } from 'querystring'
+
+export const isIndexRoute = (pathname: string): boolean => {
+  return pathname === AppRoutes.index
+}
+
+export const isNewSafeRoute = (pathname: string): boolean => {
+  return pathname === AppRoutes.newSafe.create
+}
+
+export const isWelcomeRoute = (pathname: string): boolean => {
+  return pathname === AppRoutes.welcome
+}
+
+export const isLoadSafeRoute = (pathname: string): boolean => {
+  return pathname === AppRoutes.newSafe.load
+}
```

Comment:
I think this is an overkill. They are not used anywhere other than in the side drawer. `isIndexRoute` isn't used anywhere at all according to my grep fu.

Side drawer could just do `[ AppRoutes.welcome, AppRoutes.newSafe.create, AppRoutes.newSafe.load ].includes(pathname)`